client/settings/dpcs-settingspanel/panel_evoker.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [[ "$BASH_SOURCE" == "$0" ]]
+then 
+	sudo -s python2 settingspanel.py
```

Comment:
I think storing per-user configuration is the best option here as it removes the need for sudo (and thus the script) completely. Im not sure where the new location should be though, any ideas?